hibernate-core/src/main/java/org/hibernate/sql/exec/spi/JdbcOperationQuerySelect.java

```diff
@@ -22,15 +22,15 @@
  *
  * @author Steve Ebersole
  */
-public class JdbcSelect extends AbstractJdbcOperation {
+public class JdbcOperationQuerySelect extends AbstractJdbcOperationQuery {
```

Comment:
I'm not very keen on the naming of types in this hierarchy: "jdbc operation query select" reads like four words stuck together in a random order.  And "jdbc operation query delete" is even worse.

How about `SelectQueryOperation` or something? Or just `JdbcSelectOperation`?

Reply:
I agree the namings aren't great

Reply:
So the name has "Query" in it to indicate they come from Query objects.  Not sure a better way to indicate that - maybe moving the stuff under the `query` package...